### ISSUE TITLE
NTR: MR Spectroscopy: methylhex-dienedioic acid in brain (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9037]
New term request MR Spectroscopy: methylhex-dienedioic acid in brain (elthat_200509175728): MRS

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000147
- label: Elevated methylhex-dienedioic level by MRS
- parents: [Abnormal metabolic brain imaging by MRS (HP:0012705)](https://hpo.jax.org/app/browse/term/HP:0012705)
- pmid: PMID:32027475
- def: An increase in the level of 3-methylhex-2,4-dienedioic acid or 3-methylhex-3,4-dienedioic acid in the brain identified by magnetic resonance spectroscopy (MRS).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9037
- synonyms:
